src/sys/windows_impl.rs

```diff
@@ -160,6 +165,22 @@ impl FileExt for File {
         Ok(())
     }
 
+    fn unset_sparse(&self) -> io::Result<()> {
+        let mut info: FILE_BASIC_INFO = unsafe { mem::zeroed() };
+        info.FileAttributes = 0;
```

Comment:
Can we use avoid `mem::zeroed()` please?

```suggestion
        let mut info = FILE_BASIC_INFO {
            CreationTime: 0,
            LastAccessTime: 0,
            LastWriteTime: 0,
            ChangeTime: 0,
            FileAttributes: 0,
        };
```